src/tiled/minimaprenderer.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * minimaprenderer.h
+ * Copyright 2017, Yuriy Natarov <natarur@gmail.com>
+ * Based on a minimap by Christoph Schnackenberg and Thorbjørn Lindeijer
```

Comment:
Please just copy the copyright lines from `minimap.h` instead of writing a custom message.